docs/book/getting-started/zenml-pro/self-hosted.md

```diff
@@ -1494,6 +1494,113 @@ export ZENML_PRO_API_URL=https://zenml-pro.staging.cloudinfra.zenml.io/api/v1
 zenml login
 ```
 
+## Enabling Run Templates Support
```

Comment:
Great - do we need "Day two" operations for Run templates "upgrading"

Reply:
I don't understand the question, but enabling run templates can be done as a day two operation, as explained in the new section.